test/controllers/home_controller_test.rb

```diff
@@ -1,7 +1,33 @@
 require 'test_helper'
 
-class HomeControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+class HomeControllerTest < ActionController::TestCase
+
+  test "index action returns 200" do
+    event = FactoryBot.create(:event)
+
+    get :index
+    assert_response :success
+  end
+
+  test "index action returns 200 if end_date not present" do
+    event = FactoryBot.create(:event_no_end_date)
+
+    get :index
+    assert_response :success
+  end
+
+  test "download calendar returns 200 with params" do
```

Comment:
This and the last test are probably redundant or need to be refactored.
Since that url works anyway with any parameters or even without, you are not really testing any extra functionality, unless there was a restriction on the accepted parameters (which is probably something we need to add also for security reasons)

Reply:
Ah good point @Levia! It can either accept the argument of `format == 'vcs'` or empty.